packages/manager/apps/pci-kubernetes/src/api/data/kubernetes.ts

```diff
@@ -19,21 +19,25 @@ export const getAllKube = async (projectId: string): Promise<TKube[]> => {
   return data;
 };
 
+export interface NodePool {
+  id?: string;
```

Comment:
It seems weird to have a nullable id, if the difference comes from the difference between a get and a create I suggest to create 2 different types (they can important common properties)